.github/ISSUE_TEMPLATE/Mobile_apps_bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report for Mobile Apps
+about: Create a report to help us improve the Gutenberg mobile apps version
+labels: Mobile App Android/iOS
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Tap on '....'
+3. Scroll down to '....'
+4. See '... exact error ...'
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.

```

Comment:
I've always wondered why our template has the expected behavior after the steps to reproduce. It's always felt out of place to me and that it should go right after the bug is described. With that said, the order you have here is consistent with the template we have in gb-mobile, so although I think moving this earlier would be better, I don't consider this is a required change.

Reply:
I see the order we have here (expected behavior after steps to reproduce) is also consistent with the order used by Gutenberg. In light of that, we should probably just keep the order as-is.